Readmessy (#67)
Roboy/ravestate: modules/ravestate_akinator/README.md

```
(C) Roboy 2019            
       _     _                              
      | |   (_)              _              
 _____| |  _ _ ____  _____ _| |_ ___   ____ 
(____ | |_/ ) |  _ \(____ (_   _) _ \ / ___)
/ ___ |  _ (| | | | / ___ | | || |_| | |    
\_____|_| \_)_|_| |_\_____|  \__)___/|_|    
                                                                       
```

## 20 Questions with Akinator

> The Ravestate Akinator module is now offering the 20 questions game.

Reviewer: ```suggestion
The Ravestate Akinator module offers an implementation of the 20 questions game.
```
